service/src/main/java/bio/terra/workspace/service/workspace/flight/WorkspaceFlightMapKeys.java

```diff
@@ -72,6 +70,11 @@ private ControlledResourceKeys() {}
     public static final String RESOURCE_DESCRIPTION = "resourceDescription";
     public static final String PREVIOUS_RESOURCE_DESCRIPTION = "previousResourceDescription";
     public static final String PREVIOUS_ATTRIBUTES = "previousAttributes";
+    // additional keys for job filtering
+    public static final String STEWARDSHIP_TYPE = "stewardshipType";
+    public static final String RESOURCE = "resource";
+    // this applies outside of resources. Where should it go?
+    public static final String OPERATION_TYPE = "operationType";
```

Comment:
Maybe move this to top-level `WorkspaceFlightMapKeys`? It's also used for things like "create workspace"